### ISSUE TITLE
Docker containers stopped showing with error HTTPConnection.request() got an unexpected keyword argument 'chunked'

### ISSUE THREAD

**Issue by aliencaocao** [nicolargo/glances#2614]
**Describe the bug**
```2023-12-03 22:53:45,035 -- ERROR -- containers (Docker) plugin - Can't connect to Docker (Error while fetching server API version: HTTPConnection.request() got an unexpected keyword argument 'chunked')```
This error causes running docker containers to not show up in the page.
No updates to docker.io nor glances has been done since last time it worked. Only linux kernel updates and security patches.

**To Reproduce**
1. Start Glances with no options

**Expected behavior**
It shows up

**Screenshots**

**Environement (please complete the following information)**
 - Operating System (lsb_release -a or OS name/version):
```
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.3 LTS
Release:        22.04
Codename:       jammy
```
Glances & psutil versions: Glances v3.4.0.3 with PsUtil v5.9.1
How do you install Glances (Pypi package, script, package manager, source): pip
Glances test (only available with Glances 3.1.7 or higher):
```
===============================================================================
Glances 3.4.0.3 (/usr/local/lib/python3.10/dist-packages/glances/__init__.py)
Python 3.10.12 (/usr/bin/python3)
PsUtil 5.9.1 (/usr/local/lib/python3.10/dist-packages/psutil/__init__.py)
===============================================================================
alert         [OK]    0.00002s
[]
amps          [OK]    0.00015s key=name
[{'count': 0,
 'countmax': None,
 'countmin': 1.0,
 'key': 'name',
 'name': 'Dropbox',
 'refresh': 3.0,
 'regex': True,
 'result': None,
 'timer': 2.0574214458465576}, ...]
cloud         [OK]    0.00003s
{'platform': 'OpenStack'}
connections   [NA]
containers    [OK]    0.00005s
[]
core          [OK]    0.00031s
{'log': 3, 'phys': 3}
cpu           [OK]    0.00020s
{'cpucore': 3,
'ctx_switches': 26132,
'guest': 0.0,
'guest_nice': 0.0,
'idle': 96.9,
'interrupts': 13293,
'iowait': 0.0,
'irq': 0.0,
'nice': 0.0,
'soft_interrupts': 540,
'softirq': 0.0,
'steal': 0.0,
'syscalls': 0,
'system': 0.5,
'time_since_update': 2.0551371574401855,
'total': 31.0,
'user': 2.6}
diskio        [OK]    0.00055s key=disk_name
[{'disk_name': 'sda',
 'key': 'disk_name',
 'read_bytes': 8192,
 'read_count': 1,
 'time_since_update': 2.0078208446502686,
 'write_bytes': 552960,
 'write_count': 3}, ...]
folders       [OK]    0.00003s
[]
fs            [OK]    0.00062s key=mnt_point
[{'device_name': '/dev/sda1',
 'free': 17204187136,
 'fs_type': 'ext4',
 'key': 'mnt_point',
 'mnt_point': '/',
 'percent': 64.4,
 'size': 48277495808,
 'used': 31056531456}, ...]
gpu           [OK]    0.00002s
[]
help          [OK]    0.00000s
None
ip            [NA]
irq           [NA]
load          [OK]    0.00003s
{'cpucore': 3, 'min1': 0.17138671875, 'min15': 0.02978515625, 'min5': 0.087890625}
mem           [OK]    0.00012s
{'active': 1095069696,
'available': 17220820992,
'buffers': 288755712,
'cached': 16771215360,
'free': 17220820992,
'inactive': 21937381376,
'percent': 28.5,
'shared': 7233536,
'total': 24098369536,
'used': 6877548544}
memswap       [OK]    0.00013s
{'free': 8589135872,
'percent': 0.0,
'sin': 49152,
'sout': 184320,
'time_since_update': 2.010936737060547,
'total': 8589930496,
'used': 794624}
network       [OK]    0.00037s key=interface_name
[{'alias': None,
 'cumulative_cx': 3352412,
 'cumulative_rx': 1676206,
 'cumulative_tx': 1676206,
 'cx': 1716,
 'interface_name': 'lo',
 'is_up': True,
 'key': 'interface_name',
 'rx': 858,
 'speed': 0,
 'time_since_update': 2.006051540374756,
 'tx': 858}, ...]
now           [OK]    0.00002s
'2023-12-03 22:53:47 +08'
percpu        [OK]    0.00004s key=cpu_number
[{'cpu_number': 0,
 'guest': 0.0,
 'guest_nice': 0.0,
 'idle': 25.0,
 'iowait': 2.0,
 'irq': 0.0,
 'key': 'cpu_number',
 'nice': 0.0,
 'softirq': 0.0,
 'steal': 0.0,
 'system': 3.0,
 'total': 75.0,
 'user': 14.0}, ...]
ports         [OK]    0.00000s
[{'description': 'DefaultGateway',
 'host': '10.0.0.1',
 'indice': 'port_0',
 'port': 0,
 'refresh': 30,
 'rtt_warning': None,
 'status': 0.003548,
 'timeout': 3}]
processcount  [OK]    0.02966s
{'pid_max': 0, 'running': 1, 'sleeping': 141, 'thread': 404, 'total': 200}
processlist   [OK]    0.00020s key=pid
[{'cmdline': ['java', '-Xms1024M', '-Xmx20G', '-jar', 'server.jar', 'nogui'],
 'cpu_percent': 5.4,
 'cpu_times': pcputimes(user=39934.14, system=9907.28, children_user=0.55, children_system=0.07, iowait=0.0),
 'gids': pgids(real=0, effective=0, saved=0),
 'io_counters': [180706304, 4166803456, 180706304, 4166803456, 1],
 'key': 'pid',
 'memory_info': pmem(rss=5829795840, vms=25603579904, shared=20099072, text=4096, lib=0, data=7695863808, dirty=0),
 'memory_percent': 24.1916608976014,
 'name': 'java',
 'nice': 0,
 'num_threads': 37,
 'pid': 1392,
 'status': 'S',
 'time_since_update': 2.05703067779541,
 'username': 'root'}, ...]
psutilversion [OK]    0.00003s
(5, 9, 1)
quicklook     [OK]    0.00024s
{'cpu': 31.0,
'cpu_hz': None,
'cpu_hz_current': None,
'cpu_name': '8',
'mem': 28.5,
'percpu': [{...}, {...}, {...}],
'swap': 0.0}
raid          [NA]
sensors       [OK]    0.00001s
[]
smart         [NA]
system        [OK]    0.00001s
{'hostname': 'free-arm',
'hr_name': 'Ubuntu 22.04 64bit',
'linux_distro': 'Ubuntu 22.04',
'os_name': 'Linux',
'os_version': '5.15.0-1048-oracle',
'platform': '64bit'}
uptime        [OK]    0.00008s
{'seconds': 945401}
wifi          [NA]
===============================================================================
Total time to update all stats: 0.03544s
===============================================================================
```

**Additional context**
should be caused by https://github.com/docker/docker-py/issues/3113 but it is closed so I'm not sure why am I still getting it. I am on requests >2.29

You can also [pastebin](https://pastebin.com/):

* the Glances configuration file (https://glances.readthedocs.io/en/latest/config.html#location)
* the Glances log file (https://glances.readthedocs.io/en/latest/config.html#logging)

**Comment by nicolargo** [comment]
Hi @aliencaocao 

can you give us your Python Requests and Docker lib version ?

```
$ pip freeze | grep requests
requests==2.31.0
$ pip freeze | grep docker
docker==6.1.3
```

**Comment by nicolargo** [comment]
and also URLLib ?

    $ pip freeze | grep urllib

**Comment by aliencaocao** [comment]
requests==2.31.0
docker==6.0.0
urllib3==2.1.0
No `urllib`

**Comment by aliencaocao** [comment]
So i guess its ccaused by https://github.com/nicolargo/glances/issues/2617
After downgrading to urllib3==1.26.18, my issue is solved.

**Comment by nicolargo** [comment]
Exactly !

**Comment by aliencaocao** [comment]
Updating to docker==7.0.0 solved this issue with urllib3>=2.0. Now everything is working for me with urllib3==2.2 and docker==7.0.0. I think glances should upgrade the docker required version too as urllib3<=1.26 is quite old and a lot of libraries requires >= 2.0 now.

**Comment by nicolargo** [comment]
@aliencaocao Unfortunately all Glances dependencies are not compliant with urllib > 2. See https://github.com/nicolargo/glances/pull/2397#issuecomment-1585568526
